addOns/commonlib/src/main/java/org/zaproxy/addon/commonlib/AlertTags.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Zed Attack Proxy (ZAP) and its related class files.
+ *
+ * ZAP is an HTTP/HTTPS proxy for assessing web application security.
+ *
+ * Copyright 2021 The ZAP Development Team
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.zaproxy.addon.commonlib;
+
+/** A standard set of alert tags. */
+public class AlertTags {
+
+    private AlertTags() {}
+
+    // OWASP Top 10 2021
+    public static final String OWASP_2021_A01_KEY = "OWASP-2021-A01";
+    public static final String OWASP_2021_A01_VALUE =
+            "https://owasp.org/Top10/A01_2021-Broken_Access_Control/";
+    public static final String OWASP_2021_A02_KEY = "OWASP-2021-A02";
+    public static final String OWASP_2021_A02_VALUE =
+            "https://owasp.org/Top10/A02_2021-Cryptographic_Failures/";
+    public static final String OWASP_2021_A03_KEY = "OWASP-2021-A03";
+    public static final String OWASP_2021_A03_VALUE = "https://owasp.org/Top10/A03_2021-Injection/";
```

Comment:
Since the 2021 content is still draft, it might be too early to add links.

Reply:
I'm hoping they wont change? In any case this is in an add-on so we can change them as needed :)